package.json

```diff
@@ -14,30 +14,30 @@
     },
     "license": "Apache-2.0",
     "scripts": {
-        "images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
-        "install:playground": "cd src/playground && npm install --no-package-lock",
-        "watch:sass": "sass --watch src/assets/scss:src/assets/css",
-        "watch:eleventy": "cross-env IS_DEV=true eleventy --serve --port=2022",
-        "watch:webpack": "webpack watch --mode=development",
+        "build": "npm-run-all install:playground build:sass build:website build:playground build:minify-images",
+        "build:minify-images": "imagemin '_site/assets/images' --out-dir='_site/assets/images'",
+        "build:minify-svg": "svgo -r -f ./",
+        "build:playground": "webpack --mode=production --progress",
+        "build:playground:watch": "webpack watch --mode=development",
         "build:sass": "sass --no-source-map --style=compressed src/assets/scss:src/assets/css",
-        "build:eleventy": "npx @11ty/eleventy",
-        "build:webpack": "webpack --mode=production --progress",
-        "fetch:team": "node tools/fetch-team-data.js",
+        "build:sass:watch": "sass --watch src/assets/scss:src/assets/css",
+        "build:website": "npx @11ty/eleventy",
+        "build:website:watch": "cross-env IS_DEV=true eleventy --serve --port=2022",
+        "fetch": "npm-run-all --parallel fetch:*",
```

Comment:
according to https://github.com/eslint/eslint/blob/main/docs/src/contribute/package-json-conventions.md we do not have any convention for fetch. May be we should consider adding it to the doc or replace these fetch with something like `build:fetch`?

Reply:
`build:fetch` looks good to me.

Reply:
+1

Reply:
@mdjermanovic what do you think? I could live with adding a `fetch:` prefix.

Otherwise, maybe `build:remote-data`?

Reply:
Aligned on introducing `fetch:` scripts. Lets add it to the contribute's package-json-convention also @snitin315

Reply:
I'll update the docs.